application/src/main/java/org/togetherjava/tjbot/features/chaptgpt/ChatGPTService.java

```diff
@@ -0,0 +1,69 @@
+package org.togetherjava.tjbot.features.chaptgpt;
+
+import com.theokanning.openai.OpenAiHttpException;
+import com.theokanning.openai.completion.chat.ChatCompletionRequest;
+import com.theokanning.openai.completion.chat.ChatMessage;
+import com.theokanning.openai.completion.chat.ChatMessageRole;
+import com.theokanning.openai.service.OpenAiService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.config.Config;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Service used to communicate to OpenAI API to generate responses.
+ */
+public class ChatGPTService {
+    private static final Logger logger = LoggerFactory.getLogger(ChatGPTService.class);
+    private static final long TIMEOUT_DURATION = 10000L; // In milliseconds
+    private static final int MAX_TOKENS = 3000;
+    private final OpenAiService openAiService;
+
+    /**
+     * Creates instance of ChatGPTService
+     * 
+     * @param config Config - needed for token to OpenAI API.
+     */
+    public ChatGPTService(Config config) {
+        String token = config.getOpenaiToken();
+        openAiService = new OpenAiService(token, Duration.ofMillis(TIMEOUT_DURATION));
+    }
+
+    /**
+     * Prompt ChatGPT with a question and receive a response.
+     * 
+     * @param question String - The question being asked of ChatGPT. Max is 4000 tokens.
```

Comment:
dont hardcode the tokens here, otherwise it will get outdated when someone changes the variable. instead, use `Max is {@value MAX_TOKENS} tokens`

Reply:
remove the `String - ` part (see earlier)